mk/check.mk

```diff
@@ -24,7 +24,14 @@ shellcheck:
 
 .PHONY: golangci-lint
 golangci-lint: ## Dev: Runs golangci-lint linter
-	$(GOLANGCI_LINT_DIR)/golangci-lint run --timeout=10m -v
+	$(GOLANGCI_LINT_DIR)/golangci-lint run \
+		--disable-all \
+		--enable bodyclose,contextcheck,errcheck,gci,gocritic,gofmt,gomodguard,govet,importas,ineffassign,misspell,typecheck,unconvert,unparam,whitespace \
+		--timeout=10m -v
+	$(GOLANGCI_LINT_DIR)/golangci-lint run \
+		--disable-all \
+		--enable gosimple,staticcheck,unused \
+		--timeout=10m -v
```

Comment:
Just a question out of curiosity - why we are calling it twice instead of once? Is it because of timeout?

Reply:
It's that golangci-lint runs OOM in ciecleci since `1.47.1`. I'll add a comment here as well

Reply:
got it